XCUITests/HomePageSettingsUITest.swift

```diff
@@ -167,4 +168,57 @@ class HomePageSettingsUITests: BaseTestCase {
         // There is one entry
         XCTAssertEqual(app.tables["History List"].cells.count, 3)
     }
+    
+    func testTopSitesCustomNumberOfRows() {
+        //Pre-populate top sites with 16 URLs so we can see all 4 rows
+        for i in 0..<14 {
+            pinEnoughSitesToFillFourRows(urlStr: urlList[i])
```

Comment:
I think this is not necessary, you don't have to pin the top sites so they appear, only by visiting each webpage should be enough and that's they way they should appear